### ISSUE TITLE
[xenial] scroll position not properly retained in the settings app

### ISSUE THREAD

**Issue by kugiigi** [ubports/ubuntu-touch#634]
- Device: Nexus 5
- Channel: 16.04 devel
- Build: r400

### Steps to reproduce
1. Open the settings app
2. Select an item from the bottom part
3. Go back to the main page

### Expected behavior
The scroll position should remain the same before selecting an item and after going back to the main page.

### Actual behavior
The scroll position usually resets back at the top

**Comment by Flohack74** [comment]
By settings app you mean system-settings?

**Comment by dpniel** [comment]
sounds like the currentIndex on the ListView isn't being set when selecting a list item. Should be an easy fix and a good first issue for someone :-)

**Comment by dpniel** [comment]
Oh system-settings uses just a [Flickable](https://github.com/ubports/system-settings/blob/b4bcd0f5f056fbb2286271200d6ee9ce0594a89d/src/qml/MainWindow.qml#L222) That's not so easy then :-/

**Comment by Flohack74** [comment]
I am not sure what is meant by the OP. Lets see ;)

**Comment by kugiigi** [comment]
Yes, I meant the system settings app. It's redesigned in 16.04 and knowing that it's a flickable doesn't sound good :(

**Comment by zubozrout** [comment]
Isn't it just a matter of getting contentY value and setting it to that on return?
http://doc.qt.io/qt-5/qml-qtquick-flickable.html#contentY-prop

**Comment by Flohack74** [comment]
For me it does not reset to the top but somewhere in the middle. Anyway, this I would consider cosmetic at the moment and so I moved it to enhancements.

**Comment by mariogrip** [comment]
Since this is not an regression, i'll move it to OTA-5
